lib/kitchen/driver/ec2.rb

```diff
@@ -382,9 +425,57 @@ def hostname(server, interface_type = nil)
       end
 
       def create_ec2_json(state)
-        instance.transport.connection(state).execute(
-          "sudo mkdir -p /etc/chef/ohai/hints;sudo touch /etc/chef/ohai/hints/ec2.json"
-        )
+        if windows_os?
+          cmd = "mkdir c:\\chef\\ohai\\hints; echo $null >> c:\\chef\\ohai\\hints\\ec2.json"
+        else
+          cmd = "sudo mkdir -p /etc/chef/ohai/hints;sudo touch /etc/chef/ohai/hints/ec2.json"
+        end
+        instance.transport.connection(state).execute(cmd)
+      end
+
+      def default_windows_user_data
+        #Preparing custom static admin user if we defined something other than Administrator
+        customAdminScript = ''
+        if !(instance.transport[:username] =~ /administrator/i) && instance.transport[:password]
+          customAdminScript = Kitchen::Util.outdent!(<<-EOH)
+          "Disabling Complex Passwords" >> $logfile
+          $seccfg = [IO.Path]::GetTempFileName()
+          & secedit.exe /export /cfg $seccfg >> $logfile
+          (Get-Content $seccfg) | Foreach-Object {$_ -replace "PasswordComplexity\\s*=\\s*1", "PasswordComplexity = 0"} | Set-Content $seccfg
+          & secedit.exe /configure /db $env:windir\\security\\new.sdb /cfg $seccfg /areas SECURITYPOLICY >> $logfile
+          & cp $seccfg "c:\\"
+          & del $seccfg
+          $username="#{instance.transport[:username]}"
+          $password="#{instance.transport[:password]}"
+          "Creating static user: $username" >> $logfile
+          & net.exe user /y /add $username $password >> $logfile
```

Comment:
Okay, I'm going to leave this behavior in as the default under the assumption that users won't have an image with the TK user pre-baked.  If they do, they can write their own custom user data.